content/management/configuration-api/v3.4/index.mdx

```diff
@@ -4078,6 +4078,29 @@ Informs that a user has seen events up to a specific time.
 
 </Section>
 
+# Errors
+
+<CodeResponse title={'Format of error payload'}>
+
+```json
+{
+  "error": {
+  "type": "misdirected_request",
+  "message": "Wrong region",
+  "data": {
+    // optional
+    "region": "dal"
+  }
+  }
```

Comment:
#opt
Indentation is messed up. 
Also, I'm not a fan of JSON "comments" as that's not supported by JSON - IMHO it's better to have a table with a "required" / "mandatory" column. But I also recognize this is a pattern in our docs.

Reply:
I'm adding tab (and in place where I have copied it from)